index.html

```diff
@@ -155,8 +156,9 @@
       </p>
 
       <p>
-        In ActivityPub, accounts on a server (including those of real-world users, bots,
-        and other automated processes) are represented by "<a href="#actors">actors</a>". 
+        In ActivityPub, a user is represented by "<a href="#actors">actors</a>"
+        via the user's accounts on servers. In this version of ActivityPub
```

Comment:
I'm happy with everything in this change, except could we tweak this line?  "In this version of ActivityPub" seems to imply that we'll push out a new ActivityPub revision where the actor isn't the base component, which I think (even if we have more domain modeling to do) that's going to remain the case.

How about "in this specification"?

Reply:
I deleted "In this version of ActivityPub" as you suggested. Further discussion - in issue #260